scripts/merge-script/cli.ts

```diff
@@ -30,14 +34,28 @@ async function main() {
   // Perform the merge. Force mode can be activated through a command line flag.
   // Alternatively, if the merge fails with non-fatal failures, the script
   // will prompt whether it should rerun in force mode.
-  const mergeResult = await api.merge(prNumber, force);
-
-  // Handle the result of the merge. If failures have been reported, exit
-  // the process with a non-zero exit code.
-  if (!await handleMergeResult(mergeResult, force)) {
+  if (!await performMerge(force)) {
     process.exit(1);
   }
 
+  /** Performs the merge and returns whether it was successful or not. */
+  async function performMerge(ignoreFatalErrors: boolean): Promise<boolean> {
+    try {
+      const result = await api.merge(prNumber, ignoreFatalErrors);
+      return await handleMergeResult(result, ignoreFatalErrors);
+    } catch (e) {
+      // Catch errors to the Github API for invalid requests. We want to
+      // exit the script with a better explanation of the error.
+      if (e instanceof GithubApiRequestError && e.status === 401) {
+        console.error(chalk.red('Github API request failed. ' + e.message));
+        console.error(chalk.yellow('Please ensure that your provided token is valid.'));
+        console.error(chalk.yellow(`You can generate a token here: ${GITHUB_TOKEN_GENERATE_URL}`));
```

Comment:
Should these two lines be a part of the `insufficientPermissionsToMerge` error message?  Does it make sense to include them on ever